scanner/src/lib/dataproducts/battery/ManufacturingDataSheet_v0.1.svelte

```diff
@@ -0,0 +1,153 @@
+<script lang="ts">
+  /*
+    Docs:
+
+    https://github.com/ioxio-dataspace/sandbox-definitions/blob/main/src/DigitalProductPassport/Battery/ManufacturingDataSheet_v0.1.py
+
+    https://definitions.sandbox.ioxio-dataspace.com/definitions/DigitalProductPassport/Battery/ManufacturingDataSheet_v0.1
+
+    https://gateway.sandbox.ioxio-dataspace.com/docs#/Data%20Products/DigitalProductPassport_Battery_ManufacturingDataSheet_v0_1_DigitalProductPassport_Battery_ManufacturingDataSheet_v0_1_post
+     */
+
+  import { formatNumber } from "$lib/common"
+  import DataRow from "$components/DataRow/index.svelte"
+
+  export let data: {
+    productName: string
+    batteryModel: string
+    batteryCategory: string
+    manufacturingDate: string
+    weight: number
+    capacity: number
+    power: number
+    extinguishingAgents: string[]
+    warranty: string
+    manufacturerInformation: {
+      name: string
+      postalCode: string
+      city: string
+      country: string
+      website: string
+      email: string
+    }
+    manufacturingLocation: {
+      country: string
+      city: string
+    }
+    roundTripEfficiency: {
+      initialEnergyEfficieny: number
+      degradedEnergyEfficiency: number
+    }
+    voltageLevels: {
+      nominalVoltage: number
+      maximumVoltage: number
+      minimumVoltage: number
+    }
+    temperatureRange: {
+      minimumTemperature: number
+      maximumTemperature: number
+    }
+    expectedLifetime: {
+      cycleLife: number
+      cycleRate: string
+      referenceTest: string
+    }
+    legalConformity: {
+      requirementConformity: string[]
+      conformityDeclaration: string
+    }
+  }
+
+  function camelCaseToWords(s: string) {
+    const result = s.replace(/([A-Z])/g, " $1")
+    return result.charAt(0).toUpperCase() + result.slice(1)
+  }
+</script>
+
+<article>
+  <DataRow label="Product Name" value={data.productName} />
+  <DataRow label="Battery Model" value={data.batteryModel} />
+  <DataRow label="Battery Category" value={data.batteryCategory} />
+  <DataRow label="Manufacturing Date" value={data.manufacturingDate} />
```

Comment:
Dates should also absolutely be formatted in an uniform manner using `Intl` similar to `formatNumber`. However it seems the data is not a date, but a year + month separated by `-`, and the design was implemented assuming it's a date. This needs more work to coordinate, posted on slack.